client/src/components/scorecard_page/ScorecardSummaryPanel.vue

```diff
@@ -152,20 +145,18 @@ export default defineComponent({
 <style scoped lang='scss'>
 @import 'src/assets/styles/global';
 @import '@blueconduit/copper/scss/01_settings/design-tokens';
-@import 'bulma/sass/layout/section';
-@import 'bulma/sass/elements/container';
 
-.section {
-  background-color: $navy-blue;
+.section.has-text-centered {
+  // Use a semi-opaque gradient to tint the background image for better text readability.
+  background-image: linear-gradient(rgba(4, 86, 155, 0.75), rgba(4, 86, 155, 0.75)),
+  url('~@/assets/media/understand-score@2x.png'), url('~@/assets/media/understand-score.png');
```

Comment:
(I don't see how to comment on the image files directly)

These images are pretty big (the 2x is bigger than the entire rest of the site combined). Can we use more-compressed versions? For example, this is a 300 kb JPEG of the 2x and looks identical to me 

![understand-score@2x](https://user-images.githubusercontent.com/4321880/191846322-4d4378d4-c530-4d10-bb1a-100b5191a5a5.jpg)


Also the 3x isn't referenced anywhere.

Reply:
removed in prior PR - thanks!